test/clustertest/testplugin/TestPlugin.cpp

```diff
@@ -50,6 +50,17 @@ bool BedrockPlugin_TestPlugin::peekCommand(SQLite& db, BedrockCommand& command)
             db.read(query, result);
         }
         return true;
+    } else if (command.request.methodLine == "httpstimeout") {
+        // This command doesn't actually make the connection for 35 seconds, allowing us to use it to test what happens
+        // when there's a blocking command and master needs to stand down, to verify the timeout for that works.
+        // It *does* eventually connect and return, so that we can also verify that the leftover command gets cleaned
+        // up correctly on the former master.
+        SData request("GET / HTTP/1.1");
+        request["Host"] = "www.expensify.com";
```

Comment:
can you please make this contact a host that isn't connected to our infra? We had other test that did this and we moved them to google.com because this caused tests to fail when we were in the middle of a fire which took the website down.